### DISCUSSION TITLE
Melhorias para o mini projeto de deep learning

### DISCUSSION THREAD

**Discussion by filipeas** [joaomota59/diversos#10] [Ideas] [answered]
Há algumas funções que poderiam ser adicionadas pra otimizar o uso do projeto como um mini framework para deep learning. Essa discussão foi aberta para verificarmos algumas possibilidades.

**Comment by joaomota59** [comment, marked as answer]
Revisarei o código novamente e irei verificar as modificações necessárias para a otimização do projeto.
